sql/mob_spawn_points.sql

```diff
@@ -8261,16 +8261,16 @@ INSERT INTO `mob_spawn_points` VALUES (16933038,'Flying_Spear','Flying Spear',36
 INSERT INTO `mob_spawn_points` VALUES (16933039,'Flying_Spear','Flying Spear',36,615.098,0.000,-377.595,49);
 INSERT INTO `mob_spawn_points` VALUES (16933040,'Flying_Spear','Flying Spear',36,564.009,0.000,-314.711,51);
 INSERT INTO `mob_spawn_points` VALUES (16933044,'Goobbue_Harvester','Goobbue Harvester',37,496.692,0.000,22.934,128);
-INSERT INTO `mob_spawn_points` VALUES (16933045,'Barometz','Barometz',98,465.548,-0.500,13.832,169);
-INSERT INTO `mob_spawn_points` VALUES (16933046,'Borametz','Borametz',99,448.197,-0.500,30.825,10);
+INSERT INTO `mob_spawn_points` VALUES (16933045,'Barometz','Barometz',38,465.548,-0.500,13.832,169);
+INSERT INTO `mob_spawn_points` VALUES (16933046,'Borametz','Borametz',39,448.197,-0.500,30.825,10);
 INSERT INTO `mob_spawn_points` VALUES (16933047,'Barometz','Barometz',38,459.392,-0.501,13.353,185);
 INSERT INTO `mob_spawn_points` VALUES (16933048,'Borametz','Borametz',39,437.342,-0.500,23.941,57);
 INSERT INTO `mob_spawn_points` VALUES (16933049,'Barometz','Barometz',38,469.541,-0.500,-23.776,18);
 INSERT INTO `mob_spawn_points` VALUES (16933050,'Borametz','Borametz',39,480.411,-0.501,37.913,153);
 INSERT INTO `mob_spawn_points` VALUES (16933051,'Barometz','Barometz',38,470.248,-0.500,21.134,126);
 INSERT INTO `mob_spawn_points` VALUES (16933052,'Borametz','Borametz',39,486.132,-0.501,34.731,54);
-INSERT INTO `mob_spawn_points` VALUES (16933055,'Barometz','Barometz',38,467.168,-0.500,11.415,63);
-INSERT INTO `mob_spawn_points` VALUES (16933056,'Borametz','Borametz',39,450.915,-0.500,31.892,32);
+INSERT INTO `mob_spawn_points` VALUES (16933055,'Barometz','Barometz',98,467.168,-0.500,11.415,63);
+INSERT INTO `mob_spawn_points` VALUES (16933056,'Borametz','Borametz',99,450.915,-0.500,31.892,32);
```

Comment:
Do 16933057 and 16933058 need the same change?

Reply:
So, in each of Barometz and Borametz, there is one mob with different flags (noted here by pool 98 and 99) than the others (noted here with pools 38 and 39).  In the retail capture I have from Nyu, the ID of the different mob is 16933055 for Barometz, and 16933056 for Borametz.

It might be that the "special" mob is chosen at random, each time you start a NE Apollyon run.  But unless that's the case, these values are what match the most recent retail cap.

Reply:
@cocosolos , @59blargedy ^ (we'll have to look into how the "special" mob works; random or static; re: flags)

Wren: Scripts work in Limbus branch, but nailing down the exact behavior to correct mob ID is indeed outside of scope of this PR.

Reply:
The large mobs are probably supposed to be random, but at the time I couldn't figure out a clean way to adjust flags on spawn, so just went with static for now since their functions are still random.